lib/models/stapi-model.js

```diff
@@ -1,58 +1,73 @@
     faction;
+    series;
 
     constructor(row) {
         this.id = row.id;
         this.name = row.name;
         this.species = row.species;
         this.faction = row.faction;
+        this.series = row.series;
     };
 
-    static async insert({ name, species, faction }) {
+    static async insert({ name, species, faction, series}) {
         const { rows } = await pool.query(
-            'INSERT INTO trek_characters (name, species, faction) VALUES ($1, $2, $3) RETURNING *',
-            [ name, species, faction ]
+            'INSERT INTO trek_characters (name, species, faction, series ) VALUES ($1, $2, $3, $4) RETURNING *',
+            [ name, species, faction, series ]
         );
-
+        //console.log('FIRST INSERT');
         return new stapi_model(rows[0]);
     }
 
     static async getCharacterById(id) {
         const { rows } = await pool.query(`SELECT * FROM trek_characters WHERE id=$1`, [id]);
 
+        //console.log('getCharacterById');
         return new stapi_model(rows[0]);
     }
 
     static async getAllCharacters() {
         const { rows } = await pool.query(`SELECT * FROM trek_characters`);
 
+        //console.log('getAllCharacters');
         return rows.map((row) => new stapi_model(row));
     }
 
-    static async updateCharacterById(id, { name, species, faction }) {
+    static async updateCharacterById(id, { name, species, faction, series }) {
         const existingCharacter = await stapi_model.getCharacterById(id);
         const newName = name ?? existingCharacter.name;
         const newSpecies = species ?? existingCharacter.species;
         const newFaction = faction ?? existingCharacter.faction;
+        const newSeries = series ?? existingCharacter.series;
 
         const { rows } = await pool.query(
-            `UPDATE trek_characters SET name=$1, species=$2, faction=$3 WHERE id=$4 RETURNING *`,
-            [ newName, newSpecies, newFaction, id ]
+            `UPDATE trek_characters SET name=$1, species=$2, faction=$3, series=$4 WHERE id=$5 RETURNING *`,
+            [ newName, newSpecies, newFaction, newSeries, id ]
         );
-        console.log(rows[0], 'LOOK HERE');
-
+        //console.log('updateCharacterById');
         return new stapi_model(rows[0]);
     }
 
     static async deleteCharacterById(id) {
-        const { rows } = await pool.query(
-            `DELETE FROM trek_characters WHERE id=$1 RETURNING *`, [id]);
-        
-            return new stapi_model(rows[0]);
+        const { rows } = await pool.query(`DELETE FROM trek_characters WHERE id=$1 RETURNING *`, [id]);
+            
+        console.log('********deleteCharacterById**********');
+        return new stapi_model(rows[0]);
     }
-};
+
+    static async getTrekSeries() {
+        //console.log('****** WHY WONT YOU WORK - TOP ***********')
+        const results = await request.get('http://stapi.co/api/v1/rest/season/search');
+        //console.log('****** WHY WONT YOU WORK - BOTTOM ***********')
```

Comment:
This might be my favorite set of console logs ever. If this section is still giving you problems and you want to troubleshoot let me know!